airbyte-integrations/connectors/source-google-ads/source_google_ads/streams.py

```diff
@@ -38,7 +38,15 @@ def get_query(self, stream_slice: Mapping[str, Any]) -> str:
 
     def parse_response(self, response: SearchPager, stream_slice: Optional[Mapping[str, Any]] = None) -> Iterable[Mapping]:
         for result in response:
-            yield self.google_ads_client.parse_single_result(self.get_json_schema(), result)
+            try:
+                yield self.google_ads_client.parse_single_result(self.get_json_schema(), result)
+            except Unauthenticated:
+                raise AirbyteTracedException(
```

Comment:
Would it be possible to incorporate this error handling into the `traced_exception` function located in `utils.py`? This would centralize error handling, making it more manageable and consistent across the connector.

Reply:
updated with traced_exception